far/usermenu.cpp

```diff
@@ -527,10 +527,13 @@ int UserMenu::ProcessSingleMenu(std::list<UserMenuItem>& Menu, int MenuPos, std:
 	for (;;)
 	{
 		string Names[2];
+		string Descr;
 		Global->CtrlObject->Cp()->ActivePanel()->GetCurName(Names[0], Names[1]);
+		Global->CtrlObject->Cp()->ActivePanel()->GetCurDiz(Descr);
 		const auto& strName = Names[0];
 		const auto& strShortName = Names[1];
-		const subst_context Context(strName, strShortName);
+		const auto& strDescr = Descr;
+		const subst_context Context(strName, strShortName, strDescr);
```

Comment:
There's another `subst_context` instantiation in `FillUserMenu` function, probably the description should be passed there too?
Actually, we can just pass this Context instead:
```diff
-static void FillUserMenu(VMenu2& FarUserMenu, UserMenu::menu_container& Menu, int MenuPos, fkey_to_pos_map& FuncPos, const string& Name, const string& ShortName)
+static void FillUserMenu(VMenu2& FarUserMenu, UserMenu::menu_container& Menu, int MenuPos, fkey_to_pos_map& FuncPos, const subst_context& SubstContext)
```

Reply:
I did that, see f04dbc4.